js/wavesexchange.js

```diff
@@ -282,6 +283,9 @@ module.exports = class wavesexchange extends Exchange {
                     ],
                 },
             },
+            'currencies': {
+                'WX': { 'id': 'EMAMLxDnv3xiz8RXg8Btj33jcEw3wLczL3JKYYmuubpc', 'numericId': undefined, 'code': 'WX', 'precision': 8 },
```

Comment:
This currency is retrieved by fetchMarkets and since it is the "discount fee asset" I've added it manually